Add issue templates

brunogaspar opened the pull request:
Signed-off-by: Bruno Gaspar <brunofgaspar1@gmail.com>

Niek commented:
Maybe add some instructions to get the versions: ``npm list puppeteer puppeteer-extra`` (possibly add `-json`). For Node: `node -v`.

brunogaspar commented:
Good idea @Niek

berstend commented:
Maybe not for this PR but we definitely need to mention the necessity of pasting relevant debug output (there's a bunch of recent "help recaptcha isnt' working" issues with no relevant debug info).

brunogaspar commented:
I've updated it a little bit and also added a tweaked version of @Niek suggestion.

What should we add more?

Also should we consider tweaking the GitHub labels?

brunogaspar commented on .github/ISSUE_TEMPLATE/config.yml:
```diff
@@ -0,0 +1,5 @@
+contact_links:
+  - name: Questions & Support
+    url: https://discord.gg/MUSdsS
+    about: We don't offer official support - only community support. Join our 24/7 Discord server and post your question there, we've a great community growing, that are happy to help.
```
Can be worded differently and better, please leave your suggestions :)

berstend commented:
I had some downtime and have it a shot: https://github.com/berstend/puppeteer-extra/commit/8d10f555c0af4ea206c9bab4439e93ccbba8dc40

Closing this PR as it's obsolete now. Thanks!
